docs/learn/protocols/iota2.0/core-concepts/validators.md

```diff
@@ -0,0 +1,134 @@
+
+### Decentralized Democracy
+
+There is no minimum stake requirement to become a validator in IOTA 2.0, allowing for easy participation by many nodes. In addition, users (more precisely, [accounts](mana.md#accounts)) can also participate implicitly by delegating their stake to a trusted validator, increasing their chance of being elected to the committee. This promotes a more democratic and inclusive validation process.
+
+## Validator Registration and Committee Selection Process
+
+To be included in the committee of an epoch $e$ in IOTA 2.0, a node must go through three steps:
+
+1. [Registration](#registration).
+2. [Activity](#activity).
+3. [Selection](#selection-and-voting-weight).
+
+The timeline of this process is depicted below (the actual duration of each period is not accurately depicted; instead, it depends on the value of the corresponding protocol parameters):
+
+![Timeline](/img/learn/protocols/iota2.0/core-concepts/validators/timeline.png 'An example of the timeline for the committee selection process.')
+**Image:** An example of the timeline for the committee selection process.
+
+### Registration
+
+To register as a validator for epoch $e$, an actor has to issue a block with a transaction that mutates an account to add a corresponding staking feature. Both block and transaction have to be issued before `epochStart-epochSetupDuration-activityDuration`, where `epochStart` is the beginning of epoch $e$. The registration must provide the following information:
+
+- **Account ID**: A human-readable identifier of the actor.
+- **Profit Margin**: A fixed fraction of combined validator’s and delegators’ rewards that intends to be retained by the actor.
+- **Fixed Cost**: A fixed amount deducted to cover the validator’s operating costs.
+- **Locked Funds (stake)**: A user-chosen amount of funds that will be locked.
+- **Unbonding Time**: A timestamp until the locked funds cannot be moved.
+
+The registration is only considered successful when the registration block and the mutating transaction get accepted (i.e., approved by more than $2/3$ of the current online committee) and committed to the respective slot commitment. You can set the unbonding time to "unlimited" (or a reasonably large timestamp), so you only need to register once.
+
+### Activity
+
+In the _activity period_, defined as `[epochStart-epochSetupDuration-activityDuration, epochStart-epochSetupDuration)`, an actor has to have at least one accepted block. This block can be of any type, e.g., it could be a validation block. All actors who have an accepted block within the activity period are called $`e`$-`preActive`. Note that this step needs to be done before every epoch if a validator wants to participate in the committee.
```

Comment:
```suggestion
In the _activity period_, defined as `[epochStart-epochSetupDuration-activityDuration, epochStart-epochSetupDuration)`, an actor has to have at least one accepted block. This block can be of any type, e.g., it could be a validation block. All actors who have an accepted block within the activity period are called $e$-`preActive`. Note that this step needs to be done before every epoch if a validator wants to participate in the committee.
```